commands/conversions/markdown-to-telegram.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Convert Markdown to Telegram Format
+# @raycast.mode silent
+
+# Optional parameters:
+# @raycast.icon 🔄
+# @raycast.packageName Content Tools
+# @raycast.description Convert Markdown formatting to Telegram format, excluding processing inside code blocks or quotes
+
+# Documentation:
+# @raycast.author Maxim Borzov
+
+import re
+import subprocess
+
```

Comment:
I guess we can extract the `LANG` dictionary and also the encoding from the `copy` and `paste` methods, don't you think? 
Instead of having them duplicated, lets define them outside of the methods and then just call 'em

```suggestion

env_vars = {'LANG': 'en_US.UTF-8'}
encoding = 'utf-8'
```